docs/discovery-handler-development.md

```diff
@@ -148,24 +148,10 @@ deleted Configuration.
 Rust Discovery Handler development can be kick-started using Akri's [Discovery Handler template](https://github.com/kate-goldenring/akri-discovery-handler-template) and
 [`cargo-generate`](https://github.com/cargo-generate/cargo-generate).
 
-> NOTE: `cargo-generate` depends on `openssl-dev` and `pkg-config` and need to be installed beforehand
-
-__GNU/Linux__:
-
-```sh
-apt install -y openssl-dev pkg-config
-```
-
-__macOS__:
+Install by following the [`cargo-generate` instruction](https://github.com/cargo-generate/cargo-generate#installation) and 
```

Comment:
I'd reword to clarify the two different steps: installing the tool and pulling down the template using the tool. Something like: Install [`cargo-generate`](https://github.com/cargo-generate/cargo-generate#installation) and use the tool to pull down Akri's template, specifying the name of the project with the `--name` parameter.

Reply:
resolved